src/cleanup.ts

```diff
@@ -4,9 +4,9 @@ import { Utils } from './utils';
 async function cleanup() {
     try {
         core.startGroup('Cleanup JFrog CLI servers configuration');
-        if (!Utils.addCachedCliToPath()) {
-            return;
-        }
+        const jfrogCliPath: string = core.getState(Utils.JFROG_CLI_PATH_STATE);
+        core.addPath(jfrogCliPath);
```

Comment:
Let’s handle it within a new addCachedCliToPath() function:
```ts
public static addCachedCliToPath(): boolean {
    const jfrogCliPath: string = core.getState(Utils.JFROG_CLI_PATH_STATE);
    if (!jfrogCliPath) {
        // This means that the JFrog CLI was not installed in the first place, because there was a failure in the installation step.
        return false;
    }
    core.addPath(jfrogCliPath);
    return true;
}
```